devel/ci/bodhi-ci

```diff
@@ -434,16 +437,19 @@ class Job(object):
         if self.cancelled:
             color_start = '\033[0;33m' if tty else ''
             color_end = '\033[0m' if tty else ''
-            return '{}:  {}CANCELED{}\n'.format(self.label, color_start, color_end)
+            return '{}:  {}CANCELED{}  [{}]\n'.format(self.label, color_start, color_end,
```

Comment:
Just printing the substraction will display the microseconds, won't it? Do you want to format it differently or is it fine as is?